test/helpers/createSearchClient.ts

```diff
@@ -30,7 +30,7 @@ interface Options {
  * @param options Additional customizations of the search response
```

Comment:
**question (if-minor):** Is this type any because it could be anything? Hoping to avoid type any as much as possible, and if it is needed, we should add a lint ignore comment (it wouldn't let me comment on the actual line (line 2)